[2025-05-17T13:00:00.848+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-17T13:00:00.877+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T12:00:00+00:00 [queued]>
[2025-05-17T13:00:00.889+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T12:00:00+00:00 [queued]>
[2025-05-17T13:00:00.890+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-17T13:00:00.908+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): monitor_model> on 2025-05-17 12:00:00+00:00
[2025-05-17T13:00:00.916+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9825) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-17T13:00:00.915+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_T3', 'monitor_model', 'scheduled__2025-05-17T12:00:00+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/monitor_T3.py', '--cfg-path', '/tmp/tmp_cvknqpx']
[2025-05-17T13:00:00.917+0000] {standard_task_runner.py:63} INFO - Started process 9827 to run task
[2025-05-17T13:00:00.918+0000] {standard_task_runner.py:91} INFO - Job 346: Subtask monitor_model
[2025-05-17T13:00:00.973+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_T3.monitor_model scheduled__2025-05-17T12:00:00+00:00 [running]> on host b91ac73f2b68
[2025-05-17T13:00:01.078+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_T3' AIRFLOW_CTX_TASK_ID='monitor_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-17T12:00:00+00:00'
[2025-05-17T13:00:01.079+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-17T13:00:01.128+0000] {logging_mixin.py:188} INFO - [monitor_T3] Prediction: 0.8386096954345703, Ground Truth: 0.65, Error: 0.1886096954345703
[2025-05-17T13:00:01.147+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-17T13:00:01.148+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-17T13:00:01.158+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_T3, task_id=monitor_model, execution_date=20250517T120000, start_date=20250517T130000, end_date=20250517T130001
[2025-05-17T13:00:01.212+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-17T13:00:01.255+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-17T13:00:01.258+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
